### ISSUE TITLE
Remove "Lesson" headings from each episode

### ISSUE THREAD

**Issue by tobyhodges** [datacarpentry/openrefine-socialsci#115]
The body of each episode in the lesson currently begins with a top-level heading, `# Lesson`. The lesson template adds an `h1` heading to episode pages based on the `title` field of the episode front matter (the YAML header between the two `---` lines at the start of each file). 

[Multiple `h1` headings are not good for the accessibility of the page](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/), so these extra `# Lesson` headings should be removed from each episode file.

**Comment by khalatevarun** [comment]
would like to pick this issue

**Comment by tobyhodges** [comment]
Thanks @khalatevarun. I've assigned the issue to you. Please let me know if you need any help or have any questions while you are working on it.

**Comment by khalatevarun** [comment]
sure, just wanted to confirm that this Lesson heading is to be removed, right?
Or is it expected to remove the heading tag from it and simply wrap with div tag?
![Uploading SCR-20220928-vm8.png…]()
